ls8/cpu.py

```diff
@@ -2,41 +2,78 @@
 
 import sys
 
+HLT = 1
+LDI = 130
+PRN = 71
+MUL = 162
+PUSH = 69
+POP = 70
+CALL = 80
+ADD = 160
+RET = 17
+SUB = None
+DIV = None
+
+SP = 7  # Stack Pointer
```

Comment:
Yep, this is really good. I think there's other instances where the spec designates a specific register for something? Those eat into your 8 registers, but you could also write a program that completely ignores the SP designation and uses that register for whatever you want.